expected-warnings

```diff
@@ -1,2 +1,2 @@
+"Duplicates licenses: GPL-3.0-only, GPL-3.0-or-later","Duplicates licenses: AGPL-1.0-only, AGPL-1.0-or-later","Duplicates licenses: LGPL-3.0-only, LGPL-3.0-or-later","Duplicates licenses: GFDL-1.1-or-later, GFDL-1.1-only","Duplicates licenses: LGPL-2.0-or-later, LGPL-2.0-only","Duplicates licenses: MPL-2.0, MPL-2.0-no-copyleft-exception","Duplicates licenses: GFDL-1.3-only, GFDL-1.3-or-later","Duplicates licenses: GFDL-1.2-only, GFDL-1.2-or-later","Duplicates licenses: GFDL-1.3-only, GFDL-1.3-or-later","Duplicates licenses: AGPL-3.0-only, AGPL-3.0-or-later","Duplicates licenses: GPL-1.0-or-later, GPL-1.0-only","Duplicates licenses: AGPL-1.0-or-later, AGPL-1.0-only","Duplicates licenses: AGPL-3.0-or-later, AGPL-3.0-only","Duplicates licenses: CAL-1.0, CAL-1.0-Combined-Works-Exception","Duplicates licenses: GFDL-1.2-or-later, GFDL-1.2-only","Duplicates licenses: GFDL-1.3-or-later, GFDL-1.3-only","Duplicates licenses: GPL-3.0-or-later, GPL-3.0-only","Duplicates licenses: LGPL-3.0-or-later, LGPL-3.0-only","Duplicates licenses: GPL-3.0-or-later, GPL-3.0-only","Duplicates licenses: AGPL-1.0-or-later, AGPL-1.0-only","Duplicates licenses: LGPL-3.0-or-later, LGPL-3.0-only","Duplicates licenses: GFDL-1.1-only, GFDL-1.1-or-later","Duplicates licenses: LGPL-2.0-only, LGPL-2.0-or-later","Duplicates licenses: LGPL-2.1, LGPL-2.1-only","Duplicates licenses: LGPL-2.1-only, LGPL-2.1","Duplicates licenses: MPL-2.0-no-copyleft-exception, MPL-2.0","Duplicates licenses: GFDL-1.3-or-later, GFDL-1.3-only","Duplicates licenses: GFDL-1.2-or-later, GFDL-1.2-only","Duplicates licenses: GFDL-1.3-or-later, GFDL-1.3-only","Duplicates licenses: AGPL-3.0-or-later, AGPL-3.0-only","Duplicates licenses: GPL-1.0-only, GPL-1.0-or-later","Duplicates licenses: AGPL-1.0-only, AGPL-1.0-or-later","Duplicates licenses: AGPL-3.0-only, AGPL-3.0-or-later","Duplicates licenses: GFDL-1.2-only, GFDL-1.2-or-later","Duplicates licenses: GFDL-1.3-only, GFDL-1.3-or-later","Duplicates licenses: GPL-3.0-only, GPL-3.0-or-later","Duplicates licenses: LGPL-3.0-only, LGPL-3.0-or-later","Duplicates licenses: OFL-1.1-RFN, OFL-1.1","Duplicates licenses: OFL-1.1, OFL-1.1-RFN","Duplicates licenses: OFL-1.0, OFL-1.0-no-RFN","Duplicates licenses: OFL-1.0-no-RFN, OFL-1.0","Duplicates licenses: OFL-1.0-RFN, OFL-1.0-no-RFN","Duplicates licenses: OFL-1.0-no-RFN, OFL-1.0-RFN","Duplicates licenses: OFL-1.0-RFN, OFL-1.0","Duplicates licenses: OFL-1.0, OFL-1.0-RFN","Duplicates licenses: CAL-1.0, CAL-1.0-Combined-Works-Exception"
 
```

Comment:
@VanL So here, frustratingly, I think you'll need to put the "Duplicates licenses" in both possible orders, e.g.:

"Duplicates licenses: CAL-1.0, CAL-1.0-Combined-Works-Exception","Duplicates licenses: CAL-1.0-Combined-Works-Exception, CAL-1.0"

I see that the first one appears twice in the list above, but I think the second might need to be added.

This is another one of those things where I'm sure we would improve the tooling for this if we had infinite time  :)  But try that and see if it fixes the duplicates error.